list.txt

```diff
@@ -12494,7 +12468,6 @@ glassesoutletsaleuk.co.uk
 glassesoutletuksale.co.uk
 glassworks.cf
 glavsg.ru
-gleeze.com
```

Comment:
and this one

Reply:
add new commit - where SLD domain allowed